docs/source/ruletypes.rst

```diff
@@ -5,6 +5,80 @@ Examples of several types of rule configuration can be found in the example_rule
+|      RULE TYPE                             | Any | Blacklist | Whitelist | Change | Frequency | Spike | Flatline |
++============================================+=====+===========+===========+========+===========+=======+==========+
+| ``compare_key`` (string, no default)       |     |    Req    |  Req      |    Req |           |       |          |
++--------------------------------------------+-----+-----------+-----------+--------+-----------+-------+----------+
+|``blacklist`` (list of strs, no default)    |     |   Req     |           |        |           |       |          |
++--------------------------------------------+-----+-----------+-----------+--------+-----------+-------+----------+
+|``whitelist`` (list of strs, no default)    |     |           |   Req     |        |           |       |          |
++--------------------------------------------+-----+-----------+-----------+--------+-----------+-------+----------+
+| ``ignore_null`` (boolean, no default)      |     |           |   Req     |  Req   |           |       |          |
++--------------------------------------------+-----+-----------+-----------+--------+-----------+-------+----------+
+| ``query_key`` (string, no default)         |     |           |           |   Req  |           |  Opt  |          |
++--------------------------------------------+-----+-----------+-----------+--------+-----------+-------+----------+
+| ``timeframe`` (time, no default)           |     |           |           |   Opt  |    Req    |  Req  |   Req    |
++--------------------------------------------+-----+-----------+-----------+--------+-----------+-------+----------+
+| ``num_events`` (int, no default)           |     |           |           |        |    Req    |       |          |
++--------------------------------------------+-----+-----------+-----------+--------+-----------+-------+----------+
+|``use_count_query`` (boolean, no default)   |     |           |           |        |     Opt   | Opt   | Opt      |
+|                                            |     |           |           |        |           |       |          | 
+|``doc_type`` (string, no default)           |     |           |           |        |           |       |          | 
++--------------------------------------------+-----+-----------+-----------+--------+-----------+-------+----------+
+|``use_terms_query`` (boolean, no default)   |     |           |           |        |     Opt   | Opt   |          |
+|                                            |     |           |           |        |           |       |          | 
+|``doc_type`` (string, no default)           |     |           |           |        |           |       |          | 
+|                                            |     |           |           |        |           |       |          | 
+|``query_key`` (string, no default)          |     |           |           |        |           |       |          | 
```

Comment:
Another to go under use_terms_query
``terms_size`` (Int, default 50)

This was just added to the docs in https://github.com/Yelp/elastalert/pull/84/files